mempool/policy_test.go

```diff
@@ -233,17 +233,24 @@ func TestDust(t *testing.T) {
 			true,
 		},
 		{
-			"38 byte public key script with value 5840",
-			wire.TxOut{Value: 5840, PkScript: pkScript},
+			"38 byte public key script with value 584",
+			wire.TxOut{Value: 584, PkScript: pkScript},
 			1000,
 			true,
 		},
 		{
-			"38 byte public key script with value 5850",
-			wire.TxOut{Value: 5850, PkScript: pkScript},
+			"38 byte public key script with value 585",
+			wire.TxOut{Value: 585, PkScript: pkScript},
 			1000,
 			false,
 		},
+		{
```

Comment:
add this test

Reply:
this tests errors out. possibility of new litecoin txfees causing this to be dust